CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
 <!-- insertion marker -->
+
+### Bug Fixes
+
+- Make `custom_templates` relative to MkDocs configuration file.
+
```

Comment:
Our changelog is auto-generated, can you revert this?

Reply:
Done